docs/subscriptions.md

```diff
@@ -472,3 +472,60 @@ graphql = GraphQL(
     ),
 )
 ```
+
+## Using Server-Sent Events Protocol
+
+In addition to subscriptions over WebSockets, Ariadne also allows an alternative subscriptions approach, `graphql-sse`, 
```

Comment:
This should be reworded and hard linebreak removed:

```
Instead of WebSockets, [Server-Sent Events (SSE)](https://github.com/enisdenjo/graphql-sse/blob/master/PROTOCOL.md) can be used as a transmission protocol for subscriptions. This approach uses single, long-lived HTTP connection to push data to clients.
```